frontend/packages/operator-lifecycle-manager/src/components/operator-group.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from 'react';
+import * as _ from 'lodash';
```

Comment:
Just confirming this doesn't break tree shaking? (We do have #2381 to deal with this generally.)